playbooks/common/openshift-cluster/config.yml

```diff
@@ -31,6 +31,6 @@
 
 - include: ../openshift-master/config.yml
 
-- include: additional_config.yml
-
 - include: ../openshift-node/config.yml
+
+- include: additional_config.yml
```

Comment:
There is the flannel config, the service account creation, the manageiq integraion, openshift_examples, etc.

Also, at some point we want to declare failure if < x% of nodes fail to install, at which point none of the additional_config would be completed.